app/views/_micropost.html.erb

```diff
@@ -0,0 +1,14 @@
+<li id="micropost-<%= micropost.id %>">
```

Comment:
this file should be named `app/views/microposts/_micropost.html.erb`, not `app/views/_micropost.html.erb`